src/components/UrgencyTag.jsx

```diff
@@ -0,0 +1,34 @@
+export const UrgencyTag = ({
+	color,
+	value,
+	frequency,
+	handleFrequencyChange,
+	tagName,
+}) => {
+	return (
+		<div className="h-[120px]">
+			<div
+				className={
+					color +
+					' w-32 h-72  rounded-t-full relative hover:scale-105 mx-auto -rotate-90 sm:rotate-0 border-2 border-gray-400'
```

Comment:
I think we should add an active state too. Perhaps `active:border-8`, I haven't tested it yet, but I do think it's important that it's clear which one has been selected since the radio button doesn't give a huge contrast.